frontend/src/components/atoms/hyperTexts/Badge.tsx

```diff
@@ -7,14 +7,14 @@ export enum BadgeType {
 }
 
 type Props = {
-  type?: BadgeType;
+  type?: BadgeType | null;
```

Comment:
en fait, je viens de voir que type est optionnel, donc pour moi pas besoin de mettre le null. Sry

```suggestion
  type?: BadgeType;
```